File: Class work/3-9-24/task4.c
#include<stdio.h>
main()
{
    int a[5],b[5],c[5],i;

    for(i=0;i<5;i++)
    {
        printf("enter the number a =");
        scanf("%d",&a[i]);
    }
    for(i=0;i<5;i++)
    {
        printf("enter the number b =");
        scanf("%d",&b[i]);
    }
    for(i=0;i<5;i++)
    {
       c[i]=a[i]+b[i];
    }
    for(i=0;i<5;i++)
    {
     printf("\nenter the number c = %d",c[i]);
    }
        

}

File: Class work/Python/7-12-24/task4.py
i = 1
ev = 0 
od = 0
evsum = 0
odsum = 0
while(i<=5):
    n = int(input("Enter Number: "))

    if n%2==0:
        print("Even",n)
        ev=ev+1
        evsum = evsum +n

    else:
        print("Odd",n)
        od=od+1
        odsum = odsum +n
    
    i+=1

print("Even numbers are :",ev)    
print("Odd Nuumbers are :",od)    
print("Even Sum : ",evsum)    
print("Odd Sum :",odsum)    